core/src/main/java/jenkins/model/lazy/AbstractLazyLoadRunMap.java

```diff
@@ -443,7 +455,8 @@ private Index copy() {
      * 
      * @return null if the data failed to load.
      */
-    protected R load(int n, Index editInPlace) {
+    private R load(int n, Index editInPlace) {
```

Comment:
No, it does not. Take a closer look at the signature. The `Index` parameter was a `private` class. Thus it would have been impossible for there to be any subclasses (outside of this source file). The method was only ever `protected` by accident, it seems.